ingestion/src/metadata/ingestion/source/database/snowflake/utils.py

```diff
@@ -121,13 +121,10 @@ def get_view_definition(  # pylint: disable=unused-argument
     schema = schema or self.default_schema_name
     if schema:
         cursor = connection.execute(
-            "SHOW /* sqlalchemy:get_view_definition */ VIEWS "
-            f"LIKE '{view_name}' IN {schema}"
+            f"SELECT GET_DDL('VIEW','{schema}.{view_name}') AS \"text\""
```

Comment:
should we have something like this

/* {{"app": "OpenMetadata", %%}} */

Reply:
Added it